conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+pytest.string_with_spaces = 'Variables may subsequently be rebound at any time to any object'
+pytest.string_with_comas_dots = 'It has list comprehensions, and dictionaries. Sets, and generator expressions'
+pytest.string_with_dashes = 'Python is a high-level, general-purpose programming language which is beginner-oriented'
+pytest.string_quote_parentheses_new_line = '''Python's development is conducted largely through the Python Enhancement
+                                        Proposal (PEP) process, the primary mechanism for proposing major features'''
```

Comment:
Triple quotes keeps all the spaces as they are. So the line 7 will have a lot of spaces before the Proposal word. Sometimes this is what we need. Sometimes, it can be rewritten just wrapping the long string in parenthesis. Also, if there is a single quote in a string, we can define a string with double quotes:
```suggestion
pytest.string_quote_parentheses_new_line = (
    "Python's development is conducted largely through the Python Enhancement"
    "\n  Proposal (PEP) process, the primary mechanism for proposing major features"
)
```